connectors/sync_job_runner.py

```diff
@@ -214,8 +216,30 @@ async def _execute_content_sync_job(self, job_type, bulk_options):
             sync_rules_enabled=sync_rules_enabled,
             content_extraction_enabled=content_extraction_enabled,
             options=bulk_options,
+            optimize_on_meta_timestamp=self._lookup_service_config(
+                "enable_full_sync_meta_ts_optimization", False
```

Comment:
Nit, we keep all of our default config values in `connectors/config.py`. Should this fallback be defined there?

Reply:
The fallback of `False` is defined there. I'm hesitant to put the function in `config.py` because right now we treat config as just a dict, not as a class/object. And we frequently pass around just a sub-dict. So I expect we'd end up needing to implement this outside of config.py anyway.